phpBB/phpbb/controller/helper.php

```diff
@@ -212,12 +212,29 @@ public function error($message, $code = 500)
 	public function message($message, array $parameters = array(), $title = 'INFORMATION', $code = 200)
 	{
 		array_unshift($parameters, $message);
+		$message_text = call_user_func_array(array($this->user, 'lang'), $parameters);
+		$message_title = $this->user->lang($title);
+
+		if ($this->request->is_ajax())
+		{
+			global $refresh_data;
```

Comment:
I guess we could, by creating a `redirect()` and `meta_refresh()` method in the controller helper. But that would break BC on the update from trigger_error to the controller helper way, since the old methods are no longer valid:
https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/includes/functions.php#L2478-L2487
https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/includes/functions.php#L4071-L4083